generate-config.js

```diff
@@ -136,7 +136,7 @@ const preamble = `
 version: 2.1
 
 orbs:
-  node: circleci/node@1.1
+  node: circleci/node@5.0.0
```

Comment:
Same question, why is this needed in this PR? Is this out of scope? It looks like the initial commit built successfully.

Reply:
Check the log for `included 9.4.0`, it is actually not building the image because `npx` is not found. Something changed on Circle's side w.r.t the default `machine` executor. The actual final fix is in this PR: https://github.com/cypress-io/cypress-docker-images/pull/587

Reply:
Can you make a note in the changelog summary so this information isn't lost?

Reply:
Yeah, it is testing the exit code of `npx if-docker-image-exists`, so it didn't fail the test for that command to fail.